epub33/rs/index.html

```diff
@@ -548,6 +548,15 @@ <h3>Spine</h3>
 				<p id="confreq-rs-pkg-spine-unknown" data-tests="#confreq-rs-pkg-spine-unknown">Reading Systems MUST
 					ignore all values expressed in spine <code>itemref</code>
 					<code>properties</code> attributes that they do not recognize.</p>
+
+				<p>
+					When presented with a single manifest item that is repeated multiple times in the linear flow of the spine, 
```

Comment:
Maybe replace `When presented with a single manifest item [...]` with `When presented with a single <a href="https://www.w3.org/TR/epub-33/#dfn-publication-resource">Publication Resource</a> [...]`

That should resolve the concern of @rdeltour since it won't limit this to duplicate spine references to a single item, and instead broadens the scope to resources. So it should implicitly cover two spine items with the same itemref, and two items (with different itemrefs) that resolve to the same resource.

Reply:
I like that, @bduga, I have made the change.